generator/generator_config.textproto

```diff
@@ -999,6 +999,15 @@ discovery_products {
     generate_grpc_transport: false
   }
 
+  rest_services {
```

Comment:
You need to re-merge or rebase this, right now it looks like a gigantic PR with unrelated changes.